opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -9,6 +9,7 @@
 umask 002
 MINION_HOME="/opt/minion"
 MINION_CONFIG="/opt/minion/etc/org.opennms.minion.controller.cfg"
+MINION_PROCESS_ENV_CFG="/opt/minion/etc/org.opennms.minion.process-env.cfg"
```

Comment:
The `org.opennms.minion.*.cfg` files are normally used for storing properties referenced by features i.e. https://github.com/OpenNMS/opennms/blob/develop/features/minion/dominion/grpc-client/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L20

I'd rename this to `/opt/minion/bin/minion-env.sh` or similar.

Reply:
I'd rather not add this to `bin` and not name it with `.sh` as I wanted to avoid a constraint on this file that it be sourceable/runnable. Renaming it should be fine, we could even just call it `process-env.txt`.

IMO this file can simply be a text file that we parse what we want out of.

Reply:
Initially, I had the same idea of the process environment configuration being a script that can be sourced. However, lacking OpenNMS experience, I will change the name how both of you decide.

Reply:
I was originally thinking we could keep this file more generic and it wouldn't necessarily just be for variables we want to export to the environment. Seems like we are restricting it to that use case now which I guess is fine. What we have now looks good.

Reply:
Perhaps we could call the file `minion.env`?